src/cmd/pay.rs

```diff
@@ -49,7 +52,7 @@ impl Cmd {
             fee: 0,
             payments,
             payer: keypair.public_key().to_vec(),
-            nonce: account.speculative_nonce + 1,
+            nonce: 0,
```

Comment:
I'd rather you nest the nonce assignment logic here, but that may be a personal style choice.

Reply:
More than happy to follow your lead. Should I update fee in the same way?